### PR TITLE
Fix a string that was supposed to be an f-string

### PR DESCRIPTION
**Pull request by bmerry**

### CODE REVIEW

**Review by amishatishpatel — APPROVED** [review]
Oh dear, where did this come from?

**Comment by bmerry** [comment]
> Oh dear, where did this come from?

I noticed it when running `?sensor-list`.

File: src/katsdpcontroller/tasks.py [hunk 1]
```diff
@@ -430,7 +430,7 @@ def __init__(
             self._interface_sensors[request.network] = Sensor(
                 str,
                 self.name + f".interfaces.{request.network}.name",
```
Comment (ludwigschwardt) [inline src/katsdpcontroller/tasks.py:432]: I like the mix of the new and the old :-P

Comment (amishatishpatel) [reply src/katsdpcontroller/tasks.py:432]: Dammit I didn't even notice the line above 😅

**Pull request merged**